src/utility/ExportAgent.js

```diff
@@ -0,0 +1,20 @@
+require('dotenv').config();
+const rimraf = require('rimraf');
+const Dialogflow = require('dialogflow');
+const AdmZip = require('adm-zip');
+
+const projectId = process.env.DIALOGFLOW_PROJECT_ID;
+const file = process.argv[2] || './Agent';
+
+async function exportAgent() {
+  rimraf.sync(file);
+  const agentClient = new Dialogflow.v2.AgentsClient({
+    keyFilename: process.env.GOOGLE_APPLICATION_CREDENTIALS,
```

Comment:
Isn't this done automatically? I think all google services use the `GOOGLE_APPLICATION_CREDENTIALS` env variable out of the box.

Reply:
Normally this uses the GOOGLE_APPLICATION_CREDENTIALS env from the terminal it is executed in. I explicitly wanted to override this and use the env I imported with dotenv.